rag/svr/task_executor_refactor/dataset_wiki_generator.py

```
@@ -1700,35 +1700,38 @@ async def run_wiki_incremental(
             for _doc, tid in eligible:
                 tpl = CompilationTemplateService.get_saved(tid, ctx.tenant_id)
                 cfg = (tpl.get("config") or {}) if tpl else {}
-                if isinstance(cfg, dict) and cfg.get("plan") in (True, "yes", "true"):
-                    plan = True
-                    break
+                candidate_mode = cfg.get("mode") if isinstance(cfg, dict) else None
+                if candidate_mode in ("entity", "topic"):
+                    mode = candidate_mode
         except Exception:
-            pass  # keep the handler-provided plan as fallback
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)